test/fixtures/wpt/performance-timeline/navigation-id-resource-timing.tentative.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="/common/dispatcher/dispatcher.js"></script>
+<script src="/html/browsers/browsing-the-web/back-forward-cache/resources/helper.sub.js"></script>
+<script src="navigation-id.helper.js"></script>
+<script>
+  runNavigationIdTest({
+    navigationTimes: 3,
+    testName: 'resource_timing',
+  }, "Resource Timing navigation id test");
+</script>
```

Comment:
The [original file](https://github.com/web-platform-tests/wpt/blob/master/performance-timeline/navigation-id-resource-timing.tentative.html) has an EOF, why are we amending the content?

Reply:
I didn't amend the content, I just ran the git node command. I'll run it again and see if it fixes itself

Reply:
You're right, my bad. I thought the GH web UI would show the lack of EOL (I may be misremembering things, but I believe they used to do that), but no it seems the only way to find out is to download the raw file.

> there are 14 lines

It would also show 14 lines if there were a final EOL, e.g. https://github.com/nodejs/node/blob/main/.gitattributes show 7 lines, and it has a final EOL.